ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1099,7 +1099,9 @@ static PrjFS_Result HandleFileNotification(
         notificationType,
         nullptr /* destinationRelativePath */);
     
-    if (result == PrjFS_Result_Success && placeholderFile && PrjFS_NotificationType_PreConvertToFull == notificationType)
+    if (result == PrjFS_Result_Success &&
+        placeholderFile &&
+        (PrjFS_NotificationType_PreConvertToFull == notificationType || PrjFS_NotificationType_PreDelete == notificationType))
```

Comment:
Maybe add a comment here saying that we convert to full for PreDelete because until we have WILL_RENAME support the PreDelete could be for a rename?